backend/server/controllers/courseinstances.js

```diff
@@ -551,7 +584,7 @@ module.exports = {
                 weekMaxPoints: req.body.weekMaxPoints || courseInstance.weekMaxPoints,
                 currentWeek: req.body.currentWeek || courseInstance.currentWeek,
                 finalReview: req.body.finalReview,
-                currentCodeReview: req.body.newCr.length === 0 ? '{}' : `{${req.body.newCr.join(',')}}`,
+                currentCodeReview: newCr.length === 0 ? [] : `{${newCr.join(',')}}`,
```

Comment:
curretCodeReview should be an array but `{${newCr.join(',')}}` doesn't seem to be an array

Reply:
Is this change because we updated the addon?

Reply:
Using an array caused a postgres error in Sequelize v5